### ISSUE TITLE
Ensure that the model is in evaluation mode (model.eval()) before tracing.

### ISSUE THREAD

**Issue by josiehong** [mwang87/pytorch_template#1]
I found that we need to ensure the model is in evaluation model before tracing like this: 

```
tinymodel.eval()
traced_model = torch.jit.trace(tinymodel, torch.randn(1))
torch.jit.save(traced_model, "model.pt")
```

Otherwise, there is maybe a `Tensor-likes are not close!` error.

**Comment by mwang87** [comment]
got it, added.
